lib/resources/dstu2/observation.yaml

```diff
@@ -0,0 +1,133 @@
+---
+name: Observation
+field_name_base_url: http://hl7.org/fhir/DSTU2/observation-definitions.html#Observation
+fields:
+
+- name: category
+  required: 'No'
+  cardinality: 0..1
+  type: CodeableConcept
+  description: >
+                 A code that classifies the general type of observation being made. This is used for searching, sorting
+                 and display purposes.
+  example: |
```

Comment:
We actually don't need examples for read-only resources. These are only displayed for writes.